Automattic/Gravatar-SDK-Android
# "Open settings" button in the Camera permission rationale dialog

Closes #422 

### Description

The camera permission rationale dialog lacked the option to open the App settings directly. This PR changes that. 

Unfortunately, I couldn't find a way to launch the permission settings directly.


https://github.com/user-attachments/assets/f5ff3fea-6e90-4cc2-a12a-72ba99cc4dea



### Testing Steps

1. Go to /demo-app/AndroidManifest.xml and uncomment the camera permission 
2. Start the demo app
3. Try to take a picture
4. Decline the persission
5. Rationale Dialog should be shown - tap the `open settings`
6. Confirm the Demo App settings were opened
